ci-operator/step-registry/fips-check/OWNERS

```diff
@@ -1,2 +1,7 @@
 approvers:
+- ashwindasr
+- kolyshkin
+- mrunalp
+- rhmdnd
+- rphillips
```

Comment:
I'm fine if you remove me from this list, it's been awhile since I had any interactions with openshift ci, it's probably best. Or, I can do this as a follow-up.

Reply:
I missed your comment in time, before ti's got merged.
We already have the new owners added who can now approve it,
so guess it's still fine to have yours around :) feel free to follow-up a removal in case otherwise, thanks!